changelog/fragments/phase-2-plugin-keys.yaml

```
@@ -0,0 +1,22 @@
+entries:
+  - description: >
+      (ansible/v1, helm/v1, go/v2) Added plugin keys for the `manifests` and `scorecard` phase 2 plugins
```

```suggestion
      (ansible/v1, helm/v1, go/v2) Added plugin keys for the `manifests` and `scorecard` phase 2 plugins
```

It is no following the definition described in the template. Also, note that (ansible/v1, helm/v1, go/v2) it is confuse for end-users. We have only one plugin version for Ansible and Helm. So, I think we ought to clarify that by creating 2 entries. Also, note that For Ansible/Helm based-operators it is an additional and for Go-based operators, it is a change.

---

When it comes to changelog entries, it is better to be more specific. I am following the [template](https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml#L10-L11), which says to use your suggestion if the runtime is changing, which it is not.

---

It is true.